src/components/Navbar.tsx

```diff
@@ -8,15 +12,29 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+          <NavLink
+            className={
+              ({ isActive }) => classNames('navbar-item',
+                {
+                  'has-background-grey-lighter': isActive,
+                })
+            }
+            to="/"
+          >
+            Home
+          </NavLink>
 
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+          <NavLink
+            className={
+              ({ isActive }) => classNames('navbar-item',
+                {
+                  'has-background-grey-lighter': isActive,
+                })
```

Comment:
The style definition for the `NavLink` component is duplicated and looks large, this could be moved to a separate variable and reused to improve code performance and readability.